build-system/sauce_connect/start_sauce_connect.sh

```diff
@@ -19,8 +19,12 @@
 
 CYAN() { echo -e "\033[0;36m$1\033[0m"; }
 YELLOW() { echo -e "\033[1;33m$1\033[0m"; }
+GREEN() { echo -e "\033[0;32m$1\033[0m"; }
+RED() { echo -e "\033[0;31m$1\033[0m"; }
 
 SC_VERSION="sc-4.5.3-linux"
+AUTHENTICATED_STATUS_URL="https://$SAUCE_USERNAME:$SAUCE_ACCESS_KEY@saucelabs.com/rest/v1/info/status"
```

Comment:
Is there any security concern using $SAUCE_USERNAME and $SAUCE_ACCESS_KEY here? If so, it looks like the `STATUS_URL` without any authentication also returns a JSON of the status which could suffice.

Reply:
The unauthenticated URL is rate limited. The link in the description has more details.

Reply:
https://wiki.saucelabs.com/display/DOCS/Rate+Limits+for+the+Sauce+Labs+REST+API